### ISSUE TITLE
support for profile operation

### ISSUE THREAD

**Issue by andreasherzog** [elad/node-imagemagick-native#134]
Hi, 
thank you for maintaining imagemagick-native. It works great so far. 
Currently, I am using it for resizing but wonder if you can pass in additional commands. 
Is there currently the possibility of using the profile-option of imagemagick? In particular, I would like to know how I can write IPTC metadata to a JPG image. 

Your help is very appreciated.

**Comment by krstffr** [comment]
Yes, this currently breaks my app which converts a JPG -> PNG and then the PNG throws error due to missing profile. Would be awesome!
